src/test/java/com/redhat/cloud/policies/app/TestLifecycleManager.java

```diff
@@ -55,9 +55,6 @@
     @Override
     public void stop() {
         postgreSQLContainer.stop();
-        // Helper to debug mock server issues
```

Comment:
Leave that in please. Those lines are super helpful when the mockserver does not react how you expect it.
One can then uncomment and debug.

I was always thinking to perhaps guard via command-line option, but it's probably overkill

Reply:
added again